.vsts-ci.yml

```diff
@@ -126,18 +126,18 @@ stages:
 
     - template: /eng/build.yml
       parameters:
-        agentOs: Ubuntu_18_04
+        agentOs: Ubuntu_20_04
         pool:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
             name: NetCore1ESPool-Svc-Public
-            demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+            demands: ImageOverride -equals 1es-ubuntu-2004-open
```

Comment:
@MattGal can you review the yml changes here as I'm less familiar with which images we should or shouldn't be using.